Yoast/wordpress-seo
# Set a max width on the SEMrush modal and moved to modal.css

## Context


* Sets the max width of the SEMrush modal to 712px, as per the Sketch designs, to ensure jumping the modal doesn't change in size when an alert is added / removed based on user-interaction.

## Summary


This PR can be summarized in the following changelog entry:

* Ensures SEMrush modal width doesn't change width based on changes in its content.

## Relevant technical choices:

* Moved all lines related to the modal, to `modal.css` and removed `gutenberg-modal.css` as it's no longer relevant.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and build.
* Create a new post or edit an existing one.
* Add a keyphrase and click on 'Get related keyphrases'.
* See that the modal pops up.
* Use inspector to ensure the width has the expected amount of pixels (712).
* In the inspector, remove the alert from the DOM (or extend its text).
* Notice that the modal does not change in width.

## UI changes

## Documentation

## Quality assurance


Fixes #
